tests/daikon-tests/Makefile

```diff
@@ -41,12 +41,12 @@ everything-%: quick-% nonquick-%
 	@echo ${HR}
 quick-%: SQD-% polycalc-% special-cases-% siemens-%
 	@echo ${HR}
-nonquick-%: dsaa-% mapquick-% javautil-% 6170-% large-% new-%
+nonquick-%: dsaa-% mapquick-% javautil-% 6170-% large-% new-% newRar-%
```

Comment:
Is this name from the username of the author of the commit?  Please use a better name, or eliminate it (and update line 69 below).